components/Modal/Schedule/ScheduleModal.module.scss

```
@@ -0,0 +1,91 @@
+.container {
+  width: 32rem;
+  padding: 1.5rem;
+  background-color: $white;
+  transform: translate(-50%, -50%);
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  border-radius: 16px;
+
+  .create-schedule-form {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1rem;
+    align-self: stretch;
+
+    .schedule-name {
+      display: flex;
+      padding: .5rem 0rem;
```

```suggestion
      padding: 0.5rem 0;
```
p5: 이렇게 수정해도 괜찮을 것 같아요!